Readme.md

```diff
@@ -191,7 +191,7 @@ serve --port=80
 
 You can use `--` to indicate the end of the options, and any remaining arguments will be used without being interpreted.
 
-By default options on the command line are not positional, and can be specified before or after other arguments.
+By default, options on the command line are not positional, and can be specified before or after other arguments.
```

Comment:
Musing.

I recall wondering about this and deciding to go without the comma, party because I tend to overuse the comma with complex sentences! I felt leaving the command out made the sentence easier to read without confusing the meaning.

I did a little research and including the comma does seem to be more common, and/or considered more correct.

I'll come back to this one.

----

https://www.grammarly.com/blog/comma

> When an adverbial [phrase](https://www.grammarly.com/blog/phrases/) begins a sentence, it’s often followed by a comma, but it doesn’t have to be, especially if it’s short.

Reply:
Ok, let's go with a comma. If it annoys me in some places then I could restructure the sentence. "Options on the command line are not positional by default, ...".